src/test/java/org/folio/service/file/storage/FileStorageUnitTest.java

```diff
@@ -46,6 +46,11 @@ void shouldSaveFileDataBlocking() throws IOException {
     String savedFileContent = new String(Files.readAllBytes(savedFile.toPath()));
     assertEquals(fileContent, savedFileContent);
     assertTrue(fileStorage.isFileExist(savedFileDefinition.getSourcePath()));
+    // case when path is null
+    String sourcePath = savedFileDefinition.getSourcePath();
+    savedFileDefinition.setSourcePath(null);
```

Comment:
Checking if sourcePath is empty string  could be useful here.

Reply:
Added, thank you.